third-party/easyloggingpp/src/easylogging++.cc

```diff
@@ -2398,11 +2433,25 @@ void AsyncDispatchWorker::handle(AsyncLogItem* logItem) {
 #  endif  // defined(ELPP_SYSLOG)
 }
 
+void AsyncDispatchWorker::fetchLogQueue()
```

Comment:
Add func description to explain the double buff

Reply:
done